src/OgFieldBase.php

```diff
@@ -6,9 +6,12 @@
 namespace Drupal\og;
 
 use Drupal\Core\Plugin\PluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
 
 abstract class OgFieldBase extends PluginBase implements OgFieldsInterface {
 
+  use StringTranslationTrait;
```

Comment:
I guess this isn't necessary since PluginBase does it already. :-)

Yay, tests.